external/opal-prd/hostboot-interface.h

```diff
@@ -61,6 +61,19 @@ enum MemoryError_t
 #define HBRT_CAPS_OPAL_HAS_XSCOM_RC   (1ul << 0)
 #define HBRT_CAPS_OPAL_HAS_WAKEUP_SUPPORT (1ul << 1)
 
+/**
+ * OPAL supports wakeup interface
+ */
+#define HBRT_CAPS_OPAL_HAS_WAKEUP       (1ul << 1)
+
+/**
+ * OPAL supports '2=clear all previous forces' argument
+ */
+#define HBRT_CAPS_OPAL_HAS_WAKEUP_CLEAR       (1ul << 2)
+
+/********************/
```

Comment:
If you want opal-prd to pass this capability then you have to update hservice_get_interface_capabilities().

Reply:
I deliberately didn't want to add functionality with this commit since I'm not privvy to the details.  I just wanted to get all the constants we've added in sync.